### PR TITLE
Updated Register UI

### LINKED ISSUE #1: Setup the Register UI
Need to fix the authentication UI. Setup the Register page. It should contain the following fields:
- IITK Email address 
- Password field
- Confirm Password

<br/> Make sure to setup a  function to match the two entered passwords. Also, create a dummy API function to send in the data to backend. The data object should contain the email id and the password.

### PR DESCRIPTION
**Pull request by navneet-28**
Screenshot 2021-06-13 032743
Screenshot 2021-06-13 032823
Screenshot 2021-06-13 032853

closes #1

### CODE REVIEW

**Comment by kartikcode** [comment]
Remove the welcome text in the template UI. I don't think so we need it. Instead we can have some instructions out there. Also, remove the footer as of now, apart from the copyright claims.

File: src/views/examples/Register.js [hunk 1]
```diff
@@ -1,166 +1,181 @@
...
               <FormGroup>
-                <InputGroup className="input-group-alternative mb-3">
+                <div className="text-muted text-center mt-2 mb-4">
+                  <span style={{
+                      fontWeight: 'light',
+                      color: 'red',
+                      alignItems: 'right'
+                      }}>{emailError}
+                  </span>
+                </div>
```
Comment (mridul911) [inline src/views/examples/Register.js]: Also Can you please show validations error just below given field. It is taking too much space and doesn't looks good. Same for password one.

File: src/views/examples/Register.js [hunk 2]
```diff
@@ -1,166 +1,181 @@
...
+              </FormGroup>
               <div className="text-center">
-                <Button className="mt-4" color="primary" type="button">
-                  Create account
+                <Button className="mt-4" 
+                        id="submit"
+                        disabled={isDisabled}
+                        color="primary" 
+                        type="submit"
+                        onClick={setEmailError==="valid email"? (e)=> onRegister(e) : undefined}
```
Comment (mridul911) [inline src/views/examples/Register.js]: Use this condition inside onRegister functions. Just use this onClick={(e)=> onRegister(e)} and check conditions at start of function and if false return from that function. Or you can have some disable button for this as well.

File: src/views/examples/Register.js [hunk 3]
```diff
@@ -1,166 +1,181 @@
...
+   
+    if (e) e.preventDefault();
+    try{
+      const user ={
+        email: email,
+        password: password
+      };
+      await register(user);
+      //await dispatch(register(user));
+      history.push("../../../");
```
Comment (mridul911) [inline src/views/examples/Register.js]: Please use dispatch here and ignore using such type of navigation. If user is successfully saved it automatically redirect to Admin page no need for some routing here. You can refer how login part is done.

**Review by mridul911 — CHANGES_REQUESTED** [review]

**Comment by navneet-28** [comment]
Made all the requested changes.
Screenshot 2021-06-14 130500

**Review by mridul911 — APPROVED** [review]

**Pull request merged**